config.xml

```diff
@@ -77,10 +77,16 @@
     <splash src="resources/ios/splash/Default@2x~iphone.png" width="640" height="960"/>
     <splash src="resources/ios/splash/Default~iphone.png" width="320" height="480"/>
   </platform>
+  <platform name="wp8">
```

Comment:
@adrianq Not really. ionic did it at some point (with `ionic package build android`?) and I thought it probably made sense to let them be. I'm happy to remove them otherwise.